index.js

```diff
@@ -24,7 +24,7 @@ module.exports = function updater (opts = {}) {
   // don't attempt to update during development
   if (isDev) {
     const message = 'update-electron-app config looks good; aborting updates since app is in development mode'
-    opts.logger ? opts.logger.log(message) : console.log(message)
```

Comment:
What's the purpose of removing the `console.log` here if it's gated in dev mode?

Reply:
Good catch, previously the opts.logger would always return a logger that's why I refactored it, now when we added the option to disable logger, this line will throw error complaining log doesn't exist on the object if someone set the logger to false, we should restore this line.

Reply:
maybe
```javascript
opts.logger && opts.logger.log(message)
```